.eslintrc.js

```diff
@@ -21,6 +21,17 @@
 
 const packageJson = require("./package.json");
 const isInVscode = Boolean(process.env.VSCODE_PID);
+const isInGitHubActions = Boolean(process.env.GITHUB_ACTIONS);
+
+const noCycleMaxDepth = (
+  isInVscode
+    ? 10 // very fast for editors
+    : (
+      isInGitHubActions
+        ? undefined // on CI check everything
+        : 30 // somewhat fast for local lint running
```

Comment:
nice, I was going to ask if this could be done

Reply:
I think I should reduce it a bit further